src/batch_exporter.hpp

```diff
@@ -124,6 +129,15 @@ class BatchExporter {
             attr->set_key("service.name");
             attr->mutable_value()->set_string_value(std::string(serviceName));
 
+            auto attrs = (ResourceAttr*)customResourceAttrs.elts;
+            for (ngx_uint_t i = 0; i < customResourceAttrs.nelts; i++) {
+                attr = resourceSpans->mutable_resource()->add_attributes();
+                StrView value = StrView((char*)attrs[i].value.value.data, attrs[i].value.value.len);
```

Comment:
That's not how complex values work. We can't really use complex value here, as those require client request to be evaluated.